packages/compass-connections/src/components/connection-status-notifications.tsx

```diff
@@ -134,6 +165,8 @@ const openConnectionFailedToast = (
 ) => {
   const failedToastId = connectionInfo?.id ?? 'failed';
 
+  // TODO: close the existing connection toast and make a new one for the
```

Comment:
Not happy about this idea because it will cause some weird animations if we close the connection toast and immediately open it again. Still mulling it over, might address this stacking issue in a separate PR.